libs/MeshKernel/tests/src/MeshTests.cpp

```diff
@@ -455,6 +456,7 @@ TEST(Mesh, FindEdgeCloseToAPointShouldTriggerEdgesRTreeBuild)
     auto mesh = MakeRectangularMeshForTesting(2, 2, 1.0, meshkernel::Projection::cartesian);
 
     // FindEdgeCloseToAPoint builds m_edgesRTree for searching the edges
+    mesh->BuildTree(meshkernel::Mesh::Location::Edges);
     const size_t index = mesh->FindEdgeCloseToAPoint({1.5, 1.5});
     ASSERT_TRUE(static_cast<long long>(index) >= 0); // Luca, need a better test here: ASSERT_EQ(index, actual_closest_edge_index);
```

Comment:
Are there any testing of building trees with a bounding box?

Also, are there any specific BB tests, I may have missed them